### PR TITLE
Move wow from dev-tools

### PR DESCRIPTION
**Pull request by mboukhalfa**
Refactoring dev-tools: This PR is moving part of wow folder from dev-tools to here

### CODE REVIEW

File: wow/github-workflow.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+Feature development and bug fixing are done in topic branches, branched
+of `main` branch. Upon completion and code review, topic branch is
+merged into `upstream` branch.
+
+## Branches
+
+### Topic branches (features and bug fixes)
+
+Topic branches need to be branched off `main` and named
+`type/name-username`, where  type is `feature` or `fix` and `username`
```
Comment (lentzi90) [inline wow/github-workflow.md:25]: Could we update this? I would rather see a system like `user/topic` or `user/type-topic` for the branch name.
The reason I like this style is that it is easier to identify the user when it is at the start of the name and easy to find all your own branches with autocomplete.
We are anyway not following the naming scheme so we might as well change it :upside_down_face:

**Comment by mboukhalfa** [comment]
> While moving them, we need to check if all of it is valid still. We also need to see if some of it would actually belong somewhere in the metal3-io side. For example, CI stuff is to be handled by the metal3-io CI team, not only ESJ. /hold

I am thinking anything needs access to cleura or one of the services that are private to ESJ cannot be on metal3-io the CI docs has a part involving accessing to cleura vms and managing them although the first part checking jenkins and logs can be moved to metal3-io but we end up splitting the file I guess

File: wow/github-workflow.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+Changes should be pushed to a correct origin branch:
+
+`git push -u origin <topic-branch>`
+
+You may need to force push your topic-branch, however this MUST be
+avoided as much as possible in this stage
+
+`git push -fu origin <topic-branch>`
+
+### 6. Open a Internal Pull Request (Optional)
```
Comment (lentzi90) [inline wow/github-workflow.md:123]: I think we can remove this

File: wow/github-workflow.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+`main`** branch and your commits are properly formatted.
+
+```sh
+git fetch upstream
+git checkout <topic-branch>
+git rebase upstream/main
+git push -fu origin <topic-branch>
+```
+
+### 7. Code review (Optional)
```
Comment (lentzi90) [inline wow/github-workflow.md:139]: Also this could be removed IMO

File: wow/jenkins_ci/README.md
Comment (lentzi90) [inline wow/jenkins_ci/README.md:1]: The jobs here are outdated

File: wow/github-workflow.md (hunk 1 above)
Comment (mboukhalfa) [reply wow/github-workflow.md:25]: With this will lose the history what if we merge this then made a PR to change this

Comment (lentzi90) [reply wow/github-workflow.md:25]: Sure we can update it later

**Comment by mboukhalfa** [comment]
> > While moving them, we need to check if all of it is valid still. We also need to see if some of it would actually belong somewhere in the metal3-io side. For example, CI stuff is to be handled by the metal3-io CI team, not only ESJ. /hold
> 
> I am thinking anything needs access to cleura or one of the services that are private to ESJ cannot be on metal3-io the CI docs has a part involving accessing to cleura vms and managing them although the first part checking jenkins and logs can be moved to metal3-io but we end up splitting the file I guess

@tuminoid can be unlocked ? or you want this to be split ?

**Comment by tuminoid** [comment]
We need to do the split sooner or later. The minimum requirement for this PR is that we don't move outdated information. As this is not an old study being moved for archiving, but ways of working, it should be verified that it still applies, and updated or removed if it isn't needed anymore.

**Comment by mboukhalfa** [comment]
> We need to do the split sooner or later. The minimum requirement for this PR is that we don't move outdated information. As this is not an old study being moved for archiving, but ways of working, it should be verified that it still applies, and updated or removed if it isn't needed anymore.

This should now meet the minimum requisites we can create a ticket if something need to be moved to the official docs on metal3

**Comment by tuminoid** [comment]
/approve
/unhold

Sure.

**Review by tuminoid — APPROVED** [review]

**Review by lentzi90 — APPROVED** [review]
/lgtm

**Pull request merged**